contracts/seed/Seed.sol

```diff
@@ -217,7 +221,7 @@ contract Seed {
       * @dev                     Claim locked tokens.
       * @param _locker           The address of the locker.
     */
-    function claimLock(address _locker) public allowedToClaim {
+    function claimLock(address _locker) public isActive allowedToClaim {
```

Comment:
does it mean that admin can pause the contract and block buyers from getting their token? I'm not sure that it is necessary to give this freedom to the admin